src/EditorFeatures/Core/Implementation/SmartIndent/AbstractIndentationService.cs

```diff
@@ -51,12 +55,42 @@ private IEnumerable<IFormattingRule> GetFormattingRules(Document document, int p
                 return null;
             }
 
+            return GetBlankLineIndentation(document, lineNumber, indentStyle, cancellationToken);
```

Comment:
'if' check right above this causes us to bail if we're not in the blank-line case.